### PR TITLE
Feature: Add icon plus-flash

### PR DESCRIPTION
**Pull request by allanbarcelos**
plus-flash

### CODE REVIEW

File: icons/plus-flash.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path stroke-width="1.08106" d="M8.359 11.817v7.568M4.575 15.601h7.568"/>
+    <g fill="#000" stroke="none">
```
Comment (locness3) [inline icons/plus-flash.svg]: You should use strokes, _not_ fills. Icons should always have an outline style.

**Comment by allanbarcelos** [comment]
> 
> 
> Any use case for this one ?

Yes, For  to create a button for _Add Flash_. For example:  a common _Add Button_ you would click and this will open a Add Page with all fields required or not required  but a "Add Flash" you will give to user a choose a "Add Page" with only required fields.

**Comment by locness3** [comment]
> > Any use case for this one ?
> 
> Yes, For to create a button for _Add Flash_. For example: a common _Add Button_ you would click and this will open a Add Page with all fields required or not required but a "Add Flash" you will give to user a choose a "Add Page" with only required fields.

Ok, but could you please update your PR with what i said ?

File: icons/plus-flash.svg (hunk 1 above)
Comment (allanbarcelos) [reply icons/plus-flash.svg]: Ok, I'll fix this

**Comment by allanbarcelos** [comment]
> 
> 
> That looks OK.

Ok, thanks, if it is aproved to master I'll can suggest update for https://github.com/michaelbazos/angular-feather.

**Comment by locness3** [comment]
BTW, you should add a standalone flash icon - to keep company with plus-flash.

**Comment by allanbarcelos** [comment]
> 
> 
> BTW, you should add a standalone flash icon - to keep company with plus-flash.

I dont understand, do you want I create a flash icon like this used in plus-flash?

**Comment by johnletey** [comment]
@allanbarcelos Yes, that is what @locness3 means

**Comment by allanbarcelos** [comment]
flash

**Comment by allanbarcelos** [comment]
> 
> 
> I think you shouldn't add this icon on the same PR, if you don't rename it and change its desc.

ok sorry I will revert and create a new PR for flash icon

**Comment by mittalyashu** [comment]
Even after reading this https://github.com/feathericons/feather/pull/628#issuecomment-504738104, I still don't understand the use-case of the icon.

But can you try making the flash icon small?

**Pull request closed without merging**